specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -0,0 +1,7956 @@
+        }
+      },
+      "description": "The List operation response, that contains the MongoDB collections and their properties."
+    },
+    "TableListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TableGetResults"
+          },
+          "description": "List of Table and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the Table and their properties."
+    },
+    "CassandraKeyspaceListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CassandraKeyspaceGetResults"
+          },
+          "description": "List of Cassandra keyspaces and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the Cassandra keyspaces and their properties."
+    },
+    "CassandraTableListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CassandraTableGetResults"
+          },
+          "description": "List of Cassandra tables and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the Cassandra tables and their properties."
+    },
+    "GremlinDatabaseListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GremlinDatabaseGetResults"
+          },
+          "description": "List of Gremlin databases and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the Gremlin databases and their properties."
+    },
+    "GremlinGraphListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GremlinGraphGetResults"
+          },
+          "description": "List of graphs and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the graphs and their properties."
+    },
+    "ErrorResponse": {
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      },
+      "description": "Error Response."
+    },
+    "ErrorResponseUpdatedFormat": {
+      "description": "An error response from the service.",
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "description": "Details of error response.",
+          "x-ms-external": true,
+          "properties": {
+            "code": {
```

Comment:
If you are updating this can you inherit from the common definition:  specification\common-types\resource-management\v1\types.json

Reply:
The "ErrorResponse" definition in the types.json does not correspond to the required schema documented in https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/common-api-details.md#error-response-content.

Reply:
Inherited from the types.json.